ramlfications/raml.py

```diff
@@ -76,6 +78,12 @@ class RootNode(object):
                                validator=attr.validators.instance_of(dict))
     errors           = attr.ib(repr=False)
 
+    # 1.0 attributes.
+    # @todo:
+    # eventually, we should probably make a separate class for 0.8 and 1.0
+    # as some attributes are not exclusively present in one or the other spec
+    types            = attr.ib(repr=False, validator=root_types)
```

Comment:
I think it's okay to go ahead and create a BaseRootNode object, and two child objects, one for 0.8 and one for 1.0. I'm wondering how to handle future-proofing, though... would making some "mixins" work?